app/lib/find_specific_term.rb

```diff
@@ -19,30 +19,28 @@ def self.call(*args)
 
 private
 
-  def write_csv
-    CSV.open(Rails.root.join("tmp/search_term_content_items.csv"), "wb") do |csv|
-      csv << CSV_HEADERS
+  def report
+    logger.info "Searching for #{term}..."
 
-      logger.info "Searching for #{term}..."
+    term_content_items = content_items(/#{term}/)
 
-      term_content_items = content_items(/#{term}/)
+    logger.info CONTENT_ITEM_HEADERS.join(",")
 
-      term_content_items.each do |content_item|
-        csv << csv_row(content_item)
-      end
-
-      logger.info "Found #{term_content_items.count} items containing #{term}"
+    term_content_items.each do |content_item|
+      logger.info content_item_fields(content_item).join(", ")
     end
 
+    logger.info "Found #{term_content_items.count} items containing #{term}"
+
     logger.info "Finished searching"
     # There's a subtle bug with RuboCop regexp where it thinks this
     # format of interpolated `Rails.root.join` is incorrect.
-    logger.info "CSV file at #{Rails.root.join('tmp/search_term_content_items.csv')}"
+    # logger.info "CSV file at #{Rails.root.join('tmp/search_term_content_items.csv')}"
```

Comment:
Do we still want these lines? I'm imagining so

Reply:
I actually left those comment lines in by mistake, to be honest that entire comment block and the logging is unnecessary now as a CSV is no longer being written. I'll remove it all 👍

Reply:
Could you also add an explanation into the commit message as to why we're removing the CSV writing as part of this work.

Reply:
Done 👍